### ISSUE TITLE
Angular 15 support?

### ISSUE THREAD

**Issue by ttakennn** [ngx-translate/core#1396]
When this package will support for Angular 15?

**Comment by ttakennn** [comment]
It works for me. Thanks

**Comment by kbrilla** [comment]
what we are missing is support for `provideTranslations()` etc , for apps without modules

**Comment by riderx** [comment]
made a PR for it https://github.com/ngx-translate/core/pull/1399

**Comment by BojanKogoj** [comment]
Only issue I noticed after upgrade was several failing tests (Jest)

```
NullInjectorError: R3InjectorError(DynamicTestModule)[CommonService -> TranslateService -> TranslateStore -> TranslateStore]:
   NullInjectorError: No provider for TranslateStore!
```

**Comment by poojakumari19** [comment]
> Only issue I noticed after upgrade was several failing tests (Jest)
> 
> ```
> NullInjectorError: R3InjectorError(DynamicTestModule)[CommonService -> TranslateService -> TranslateStore -> TranslateStore]:
>    NullInjectorError: No provider for TranslateStore!
> ```

@BojanKogoj  Do you get any solution for this and also I am facing this issue **The pipe 'translate' could not be found in the 'MessageBoxComponent' component. Verify that it is declared or imported in this module. Find more at https://angular.io/errors/NG0302** .

**Comment by BojanKogoj** [comment]
Well it was in tests, so I just mock and provided what was needed. Not sure why it broke now at all.

**Comment by AvniAnay** [comment]
> poojakumari19

I had the similar issue , I had to add TranslateModule to all child components(standalone components)

**Comment by poojakumari19** [comment]
> 

it worked for me now.Thanks

**Comment by doughlass** [comment]
Any update on this ticket? Would be nice to get the dependencies updated to the latest.

**Comment by kbrilla** [comment]
Nah, moved to transloco myself. They have migration to ease the process

W dniu czw., 4.05.2023 o 18:32 doughlass ***@***.***>
napisał(a):
